department-of-veterans-affairs/va.gov-cms
# VACMS-9330: Update template to include fallback for non-video media e…

…mbeds.

## Description

closes #9330 

## Testing done
manual

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/172201195-4f0b9790-6db4-4c40-a4a5-72ee27f82313.png)


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As a CMS user
1. View a KB article page with an embedded image (`help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/best-practices-for-vamc-system-banner-alerts`)
   - [x] Validate that an image is visible on the page
2. Edit a KB article and add an embedded video (`help/vamc/about-locations-content-for-vamcs/about-alerts-and-operating-statuses/how-to-edit-covid-19-status-for-a-vamc-facility`)
   - [x] Validate the video title/duration appear above the video

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
